moped-database/metadata/tables.yaml

```diff
@@ -245,80 +245,102 @@
     - role: moped-admin
       permission:
         check: {}
+        set:
+          created_by_user_id: x-hasura-user-db-id
+          updated_by_user_id: x-hasura-user-db-id
         columns:
-          - project_extent_id
-          - source_layer
           - component_id
-          - id
+          - created_by_user_id
```

Comment:
This one permission snuck in for insert `created_by_user_id` by moped-admin in `feature_drawn_lines`

Reply:
Thank you Mike! I appreciate you catching that! I must have miss-clicked in the console UI when I was working on the permissions this morning. Thanks again! ca2ba5d03f62dac1e8fc287ccfb14fb703b3c5e7

Reply:
🙌🙏🚀  100% - there are tons of boxes to check on this one 😵‍💫